lib/utils/forkedblockchain.js

```diff
@@ -60,21 +60,15 @@ ForkedBlockchain.prototype.initialize = function(accounts, callback) {
 
     self.forkVersion = version;
 
-    BlockchainDouble.prototype.initialize.call(self, accounts, function(err) {
-      if (err) {
-        return callback(err);
-      }
-
-      self.patchVM(self.vm);
-
-      callback();
-    });
+    BlockchainDouble.prototype.initialize.call(self, accounts, callback);
```

Comment:
We don't need to call `patchVm` since it already gets called within `createVMFromStateTrie`.